code/modules/projectiles/gun_attachables.dm

```diff
@@ -1171,46 +1202,6 @@ Defined in conflicts.dm of the #defines folder.
 	select_gamemode_skin(type)
 	attach_icon = icon_state
 
-/obj/item/attachable/scope/mini_iff
```

Comment:
I would personally leave this in for potential event use, but that's up to you.

Reply:
I had originally intended to do so, but I found that in practice a similar amount of varediting is needed to just make a gun IFF as to spawn in an un-obtainable item and varedit the desired guns to accept it, I decided it was not worthwhile to maintain the code for this niche use case as a result.